socs/agents/acu/agent.py

```diff
@@ -85,6 +93,18 @@ def __init__(self, agent, acu_config='guess', exercise_plan=None,
         self.scan_params = {}
         self._set_default_scan_params()
 
+        startup_idle_reset = (self.acu_config['platform'] in ['lat', 'ccat']
+                              and not disable_idle_reset)
+
+        if isinstance(ignore_axes, str):
+            ignore_axes = [x.strip() for x in ignore_axes.split(',')]
+        if ignore_axes is None:
+            ignore_axes = []
+        assert all([x in ['az', 'el', 'third'] for x in ignore_axes])
```

Comment:
If `ignore_axes` isn't restricted in the argparse, perhaps put this in a try/except to catch the `AssertionError` and reraise with a useful error like `f"One or more of {ignore_axes} not in ['az', 'el', 'third']"`.